index.html

```diff
@@ -1,8 +1,6 @@
 <!DOCTYPE html>
-<!--[if lt IE 8]><html class="lt-ie10 lt-ie9 lt-ie8" lang="nl"><![endif]-->
-<!--[if IE 8]><html class="lt-ie10 lt-ie9" lang="nl"><![endif]-->
-<!--[if IE 9]><html class="lt-ie10" lang="nl"><![endif]-->
-<!--[if gt IE 9]><!--><html lang="nl"><!--<![endif]-->
+<!--[if lt IE 10]><html class="lt-ie10" lang="nl"><![endif]-->

```

Comment:
Toevallig vandaag nog gebruikt, ondanks dat we hem niet meer ondersteunen was het met hele eenvoudige aanpassingen alsnog makkelijk om alsnog IE9 te verbeteren.
Maar eens, zou er standaard wel nog uit mogen om hem zelf toe te voegen waar nodig.